README.md

```diff
@@ -25,14 +25,15 @@ The client is configured entirely through environment variables. These are liste
 - `CONJUR_AUTHN_URL`: URL pointing to authenticator service endpoint
 - `CONJUR_AUTHN_LOGIN`: Host login for pod e.g. `namespace/service_account/some_service_account`
 - `CONJUR_SSL_CERTIFICATE`: Public SSL cert for Conjur connection
+- `CONJUR_TOKEN_TIMEOUT`: Timeout for fetching a new token (defaults to 6 minutes). In most cases, this variable shoudl not be modified.
```

Comment:
shoudl => should

Reply:
I think it's also worth mentioning when and why this parameter should be used. Just to be as clear as possible to the end user.